Enterprise-CMCS/eAPD
# dmirano/4513-activities-dashboard-help-text

Resolves #4513 

### Description
Changes the help text on the activities dashboard for MMIS APDs

### Significant changes or possible side effects
Only files dealing with the Activities Dashboard was changed

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A HITECH APD | On the Activities Dashboard | The old help text appears | Jest | 
| A MMIS APD | On the Activities Dashboard | The new help text appears | Jest | 

### Steps to manually verify this change

1. Go to the Activities Dashboard and verify the help text was changed to the correct text
2. Verify the help text is only for MMIS

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
